src/radians.rs

```diff
@@ -35,8 +35,8 @@ impl Radians {
         self.value
     }
 
-    /// Try getting the radians value of the degrees
-    pub fn try_from_degrees(degrees: f64) -> Result<Radians, RadiansError> {
+    /// Convert degrees to radians
```

Comment:
Add `## Errors` and `## Examples` sections

Reply:
Resolved in 3569e7691b9d91c2eb7d6762d6ccc76d487d8f01

Reply:
Use [std::f64::consts::FRAC_PI_2](https://doc.rust-lang.org/std/f64/consts/constant.FRAC_PI_2.html) instead of `PI / 2.0`